### PR TITLE
[WAL-944] Reorganization

### PR DESCRIPTION
**Pull request by Ifropc**
All functions with SEP-24 functionality has been moved to `Interactive` class (responsible for SEP-24)
Also, added SEP12 alias for Customer class/function

### CODE REVIEW

File: wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt [hunk 1]
```diff
@@ -266,13 +173,19 @@ internal constructor(
   }
 }
 
+suspend fun Anchor.sep12(authToken: AuthToken): Sep12 {
+  return this.customer(authToken)
+}
+
 fun Anchor.sep24(): Sep24 {
   return this.interactive()
 }
```
Comment (JakeUrban) [inline wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt:182]: Why do we encourage the use of `anchor.interactive()` vs `anchor.sep24()`? The latter seems to be more straightforward, and makes it easy to add more SEPs without defining some one-word alias to it.

Comment (Ifropc) [reply wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt:182]: It's just how it's been historically. I can rename it to sep24 and keep current alias there

**Review by JakeUrban — APPROVED** [review]

File: wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt (hunk 1 above)
Comment (CassioMG) [reply wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt:182]: That's a good point, I'm also in favor of using `sep24()`. Could we please also rename the `Interactive` class to `Sep24` instead? So it's clear that it's the class that interfaces with all Sep-24 related stuff - what do you think?

Comment (Ifropc) [reply wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt:182]: Yep that's the idea 👍🏼

**Review by CassioMG — APPROVED** [review]

**Pull request merged**